third_party/blink/renderer/core/css/css_font_selector.cc

```diff
@@ -68,12 +69,20 @@ UseCounter* CSSFontSelector::GetUseCounter() const {
 void CSSFontSelector::RegisterForInvalidationCallbacks(
     FontSelectorClient* client) {
   CHECK(client);
+  recordreplay::Assert("[RUN-1219-1728] CSSFontSelector::RegisterForInvalidationCallbacks");
   clients_.insert(client);
+  if (recordreplay::IsRecordingOrReplaying("RUN-1219-CSSFontSelector")) {
```

Comment:
This requires a change to `v8`'s `api.cc`: `gKnownFeatures` (or what it's called?)
I'd recommend using the general-purpose label that already exists.
→ [Reference PR](https://github.com/replayio/chromium/pull/371/files) for making these types of changes.

Reply:
Was `"avoid-weak-pointers"` already added before that PR?

Reply:
Yeah it's there on v8's tip so I'll just use that.